src/__experimental__/components/date/date.stories.js

```diff
@@ -33,9 +34,8 @@ const setValue = (ev) => {
   store.set({ value: ev.target.value });
 };
 
-storiesOf('Experimental/Date Input', module)
-  .addDecorator(StateDecorator(store))
-  .add('default', () => {
+function makeStandardStory(name, themeSelector) {
```

Comment:
Why U R using two types (different naming) for `makeStory` and `makeStandardStory`? I mean we could use `makeStory` for all classic/default stories (to make the naming standardized) and for Validation/Multiple we could use `make${Validation/Multiple}Story`? What do U think about that?

Reply:
Yes, that's a good idea for consistency / standardised naming. Renamed in c3112ea :thumbsup: